.github/workflows/ci.yml

```diff
@@ -68,6 +68,31 @@ jobs:
           suite: ${{ matrix.suite }}
           os: ${{ matrix.os }}
 
+  lua_test:
+    needs: [mdl, yamllint, delivery]
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        os:
+          - 'centos-7'
+        suite:
+          - 'source-lua'
```

Comment:
any reason not to add this test to the suites on line 49-54?

Reply:
the lua installation steps in `source-lua` are not platform agnostic. It seemed like it would get run against every platform in the matrix

Reply:
Ah - makes sense. What do you think about adding platform checks in the source-lua to handle the different paths, packages, etc.?

Reply:
We'll keep it as is. No need to handle installing lua on all platforms to test it out